src/databricks/labs/ucx/source_code/known.json

```diff
@@ -3508,6 +3508,20 @@
   "dill": {
     "dill": []
   },
+  "diskcache": {
+    "diskcache": [],
+    "diskcache.cli": [],
+    "diskcache.core": [],
+    "diskcache.djangocache": [],
+    "diskcache.fanout": [
+      {
+        "code": "direct-filesystem-access",
+        "message": "The use of direct filesystem references is deprecated: /"
```

Comment:
is it using spark?

Reply:
Did not find spark references in codebase